aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/CreateHandler.java

```diff
@@ -73,6 +75,18 @@ private ProgressEvent<ResourceModel, CallbackContext> createDbClusterParameterGr
                 });
     }
 
+    private ProgressEvent<ResourceModel, CallbackContext> createDbClusterPGWithAllTagsOrSystemTags(final AmazonWebServicesClientProxy proxy,
+                                                                                        final ProxyClient<RdsClient> proxyClient,
+                                                                                        final ProgressEvent<ResourceModel, CallbackContext> progress,
+                                                                                        final Tagging.TagSet allTags) {
+        ProgressEvent<ResourceModel, CallbackContext> progressEvent = createDbClusterParameterGroup(proxy, proxyClient, progress, allTags);
+        if(HandlerErrorCode.AccessDenied.equals(progressEvent.getErrorCode())) { //Resource is subject to soft fail on stack level tags.
+            Tagging.TagSet systemTags = Tagging.TagSet.builder().systemTags(allTags.getSystemTags()).build();
+            return createDbClusterParameterGroup(proxy, proxyClient, progress, systemTags);
```

Comment:
nit: Here is an idea: `createDbClusterParameterGroup` might set a flag like: `taggingOk: true`, which would be checked by `updateTags` and skip this step if it was an optimistic tagging case.